File: Modul 3. Kondisional/PRAK304/PRAK304-2310817120003-NatalieGraceKatiandagho.py
x = int(input())

if x == 0:
    print('Nol')
elif x > 0 and x < 10:
    print('Satuan')
elif x >= 10 and x < 20:
    print('Belasan')
elif x >= 20 and x < 100:
    print('Puluhan')
else:
    print('Anda Menginput Melebihi Limit Bilangan')

File: Modul 3. Kondisional/PRAK304/PRAK304-2310817120003-NatalieGraceKatiandagho.c
#include <stdio.h>
void main()
{
    int x;
    scanf("%d", &x);
    if (x == 0)
    {
        printf("Nol");
    }
    else if (x > 0 && x < 10)
    {
        printf("Satuan");
    }
    else if (x >= 10 && x < 20)
    {
        printf("Belasan");
    }
    else if (x >= 20 && x < 100)
    {
        printf("Puluhan");
    }
    else
    {
        printf("Anda Menginput Melebihi Limit Bilangan");
    }
}